packages/commerce-sdk-react/src/constant.ts

```diff
@@ -14,6 +14,12 @@ export const IFRAME_HOST_ALLOW_LIST = Object.freeze([
     'https://runtime-admin-preview.mobify-storefront.com'
 ])
 
+// We hardcode these here since we don't want commerce-sdk-react to have a dependency on pwa-kit-runtime
+export const MOBIFY_PATH = '/mobify'
```

Comment:
I think we previously may have had a discussion about changing the name of this variable. It's 2024 we should definitely not be using "mobify" for any new variables in our code.

Reply:
How are we expecting that this value is configured. I imagine that we would require this to be configurable if it was changed in the rest of the application?

Reply:
Follow up discussion happened in Slack convos but to summarize, we will not make the '/mobify' part configurable for now as there are places in MRT that also are looking explicitly for the '/mobify' path